docs/modules/ROOT/pages/reference/extensions/js-dsl.adoc

```diff
@@ -0,0 +1,27 @@
+// Do not edit directly!
+// This file was generated by camel-quarkus-maven-plugin:update-extension-doc-page
+= JavaScript DSL
+:cq-artifact-id: camel-quarkus-js-dsl
+:cq-native-supported: true
+:cq-status: Stable
+:cq-description: An YAML stack for parsing YAML route definitions
```

Comment:
`An YAML stack for parsing YAML route definitions` does not sound right.

Reply:
yah, forgot to regen